src/elements/content-explorer/ItemGridCell.js

```diff
@@ -24,10 +26,20 @@ const ItemGridCell = ({
     view,
     ...rest
 }: Props) => {
+    const url = getProp(item, `thumbnailUrl`);
+    const itemClassName = classNames(
+        'bce-ItemGridCell-item',
+        { 'bce-ItemGridCell-item--postLoadThumbnail': url },
```

Comment:
Modifiers (`--modifier`) should be used for variants of a component, but these look more like descendent component names. You could also consider casting `url` to a boolean via `!!`.